bastion.tf

```diff
@@ -0,0 +1,155 @@
+resource "aws_security_group" "win_bastion" {
+  name        = "win-bastion"
+  description = "Win Bastion SG"
+  vpc_id      = module.vpc.vpc.id
+  tags        = local.common_tags
+}
+
+resource "aws_security_group_rule" "win_outbound" {
```

Comment:
you could simplify these by iterating over a list of maps. There's an example here https://github.com/dwp/dataworks-aws-ingestion-ecs-cluster/blob/master/security_groups.tf#L2